keyboards/meson/config.h

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2019 Luciano M
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Luciano M
+#define PRODUCT         Meson
+#define DESCRIPTION     3x14 split keyboard with two thumb keys per side
+
+/* key matrix size */
+#define MATRIX_ROWS 8 // 4 rows each half
+#define MATRIX_COLS 7
+
+#define MATRIX_ROW_PINS { F7, C6, F6, F5 }
+#define MATRIX_COL_PINS { D4, D7, E6, B3, B2, B6, F4 }
+// #define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+#define USE_I2C
+#define SOFT_SERIAL_PIN D0
+#define MASTER_LEFT
```

Comment:
Actually, you don't technically need this. It's the default.   

And since that's the case, you're forcing users to undefine it, first. 

```suggestion
```